README.md

```diff
@@ -174,28 +181,38 @@ isomorphicRoutes(app, {
 
    Example of the script:
 
-   ```
-       importScripts("https://www.gstatic.com/firebasejs/4.8.1/firebase-app.js");
-       importScripts("https://www.gstatic.com/firebasejs/4.8.1/firebase-messaging.js");
-       firebase.initializeApp({
-         messagingSenderId: <your message sender Id>
-       });
-       const messaging = firebase.messaging();
-       function messageHandler(payload) {
-         const data = payload["data"];
-
-         var notificationTitle = data.title;
-         var notificationOptions = {
-           body: data.body,
-           icon: data["hero_image_s3_url"],
-           image: data["hero_image_s3_url"],
-           data: data
-         };
-
-         return self.registration.showNotification(notificationTitle,
-           notificationOptions);
-       }
-       messaging.setBackgroundMessageHandler(messageHandler);
+   ```js
+    importScripts("https://www.gstatic.com/firebasejs/9.6.10/firebase-app-compat.js");
+    importScripts("https://www.gstatic.com/firebasejs/9.6.10/firebase-messaging-compat.js");
+
+    const firebaseConfig =  <%- JSON.stringify(firebaseConfig ) %>
+    firebase.initializeApp(firebaseConfig);
+    self.addEventListener('notificationclick', (event) => {
+      const url = get(event, ["notification", "data", "url"]);
```

Comment:
@Jeevan-Kishore lodash get won't be defined here right?

Reply:
Lets use `||` and error safe it